wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/RoboRioSim.java

```diff
@@ -15,14 +15,19 @@
  */
 @SuppressWarnings({"PMD.ExcessivePublicCount", "PMD.TooManyMethods", "PMD.UseUtilityClass"})
 public class RoboRioSim {
+  @SuppressWarnings("AbbreviationAsWordInName")
   public static CallbackStore registerFPGAButtonCallback(NotifyCallback callback,
                                                          boolean initialNotify) {
     int uid = RoboRioDataJNI.registerFPGAButtonCallback(callback, initialNotify);
     return new CallbackStore(uid, RoboRioDataJNI::cancelFPGAButtonCallback);
   }
+
+  @SuppressWarnings("AbbreviationAsWordInName")
   public static boolean getFPGAButton() {
     return RoboRioDataJNI.getFPGAButton();
   }
+
+  @SuppressWarnings({"AbbreviationAsWordInName", "ParameterName"})
   public static void setFPGAButton(boolean fPGAButton) {
```

Comment:
Here I would rename the parameter to be `fpgaButton` and remove the suppressions you added.